src/main/java/edu/cornell/kfs/coa/batch/CuCoaBatchConstants.java

```diff
@@ -0,0 +1,29 @@
+package edu.cornell.kfs.coa.batch;
+
+import java.sql.Date;
+
+public class CuCoaBatchConstants {
+    
+    public static final String CSV_FILE_EXTENSION = ".csv";
```

Comment:
Can we use the constant for CSV file extension in KFSConstants?

Reply:
I spoke with Nancy and we are going to create a CUKFSConstants.FileExtension class add it there